content/applications/sales/subscriptions/scheduled_actions.rst

```diff
@@ -0,0 +1,174 @@
+=================
+Scheduled actions
+=================
+
+In Odoo *Subscriptions*, there are two scheduled actions that manage subscriptions. They are turned
+on, by default, so keep that in mind while using the *Subscriptions* application. They can be
+deactivated at any time, if needed, to manage subscriptions manually.
+
+Access scheduled actions
+========================
+
+.. important::
+   To access these scheduled actions, *Developer Mode* **must** be activated.
+
+   To activate developer mode, navigate to :menuselection:`Settings app --> Scroll down to Developer
+   Tools section --> Activate the developer mode`.
+
+With developer mode activated, navigate to :menuselection:`Settings app --> Technical --> Scheduled
+Actions`.
+
+.. image:: scheduled_actions/scheduled-actions-technical-settings-page.png
+   :align: center
+   :alt: The scheduled actions option under the technical menu in the Odoo Settings application.
+
+Doing so reveals a dedicated :guilabel:`Scheduled Actions` page. On this page, there is a complete
+list of scheduled actions for the entire database.
+
+From here, enter `Subscription` in the search bar. Doing so provides three subscription-specific
+results. The following documentation focuses on the last two results in the list:
+
+- :guilabel:`Sale Subscription: generate recurring invoices and payments`
+- :guilabel:`Sale Subscription: subscriptions expiration`
+
+.. image:: scheduled_actions/scheduled-actions-page-subscription-results.png
+   :align: center
+   :alt: The subscription-related results on the scheduled actions page in Odoo Settings.
+
+Ensure the scheduled actions are :guilabel:`Active`. If there is a ticked checkbox in the
+:guilabel:`Active` column for the desired subscription-related scheduled action on the
+:guilabel:`Scheduled Actions` page, that means they are `Active`.
+
+If a scheduled action needs to be activated, click into the desired scheduled action from the list.
+
+.. image:: scheduled_actions/scheduled-action-form.png
+   :alt: The scheduled action form in the Odoo Settings application.
+
+Then, from the scheduled action form, toggle the switch in the :guilabel:`Active` field to the
+right. Doing so turns the switch green, indicating that the scheduled action is now `Active`.
+
+The ability to set up how often the scheduled action runs is also available on the scheduled action
```

Comment:
Might be nice to  include what a scheduled action actually is, and link to any existing documentation on it...